site/src/docs/configuration/env-vars.md

```diff
@@ -152,6 +152,9 @@ Nextest also sets these environment variables at runtime, matching the behavior
 
 Additionally, environment variables specified in [the `[env]` section of `.cargo/config.toml`](https://doc.rust-lang.org/cargo/reference/config.html#env) are also set.
 
+Since `cargo test` also sets variables specified in the build script via [`cargo::rustc-env`](https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-env) at test runtime, `nextest` does the same.
+However, note that [`cargo` discourages using these variables at runtime](https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-env).
```

Comment:
Thanks for adding this! I generally add documentation at the time the release happens, so that the nexte.st site is up-to-date. I'll remove this for now and will re-add it when I'm getting ready to release 0.9.82 (hopefully in the next 1-2 days).